autoscheduler/frontend/src/components/SchedulingPage/SchedulingPage.tsx

```diff
@@ -36,8 +44,9 @@ const SchedulingPage: React.FC<SchedulingPageProps> = ({
   }, [dispatch, termCurr]);
 
   return (
-    <div className={styles.pageContainer}>
-      <div className={styles.leftContainer}>
+    <div className={styles.pageContainer} style={fullscreen ? { justifyContent: 'flex-end' } : null}>
+      {/* Hide the left & middle column if we're fullscreen */}
+      <div className={styles.leftContainer} style={fullscreen ? { display: 'none' } : null}>
```

Comment:
I don't like using style here, use additional classes instead (same applies to NavBar)